var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -136,90 +133,119 @@ class Llvm(CMakePackage, CudaPackage, LlvmDetection, CompilerPackage):
     conflicts("+flang", when="@:10")
     conflicts("+flang", when="~clang")
 
-    variant("lldb", default=True, description="Build the LLVM debugger")
+    # lldb
+    variant("lldb", default=False, description="Build the LLVM debugger")
     conflicts("+lldb", when="~clang")
 
-    variant("lld", default=True, description="Build the LLVM linker")
+    # lld
+    variant("lld", default=False, description="Build the LLVM linker")
```

Comment:
Most of these I really don't mind, lldb is heavy and polly can be too, but this one is throwing the baby out with the bathwater.  Like openmp it takes a few seconds to build, and it makes the links of runtime components faster by a non-negligible amount, as well as links with the resulting stack.  Is there an issue you're working around related to lld?